.github/workflows/main.yml

```diff
@@ -178,7 +178,7 @@ jobs:
   
   react-native:
     name: Check react native
-    runs-on: macOS-latest
+    runs-on: macOS-13
```

Comment:
The RN / `ktlint` issue was solved with #762.

No need to change the macOS version anymore, just rebase on latest master and it should work.

Reply:
Thanks @ok300! Rebased to master